localization/english/culture/wc_basic_innovations_l_english.yml

```
@@ -0,0 +1,27 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ innovation_animal_war_training:0 "Animal War Training"
+ innovation_animal_war_training_desc:0 "Through skills taught to us my the best beastmasters, we can communicate, train and discipline the animals around us."
+ 
+ innovation_rogues_guild:0 "Rogues Guild"
+ innovation_rogues_guild_desc:0 "."
```

Just thought I'd throw a suggestion out for this one.
```suggestion
 innovation_rogues_guild_desc:0 "Guilds made up of rogues, thieves, and assassins are a presence in many major cities, though rarely officially acknowledged. Despite that, however, their unique talents, blending stealth, skill, and ruthlessness, make them a formidable asset for any nation."
```